elements/rh-cta/docs/30-code.md

```diff
@@ -0,0 +1,10 @@
+{% renderOverview %}{% endrenderOverview %}
```

Comment:
"Cta" `h1` heading is dusplicated

Reply:
needs to be fixed in pfe-tools

Reply:
https://github.com/patternfly/patternfly-elements/issues/2437